docs/tutorials/Cargo.toml

```diff
@@ -1,16 +0,0 @@
-# This file is part of ICU4X. For terms of use, please see the file
-# called LICENSE at the top level of the ICU4X source tree
-# (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
-
-# Not part of a tutorial. This workspace is for testing only.
-
-[workspace]
```

Comment:
Observation: putting everything in its own workspace, instead of the tutorials sharing a workspace, means we end up with more target directories, right? How much of a toll does that take on disk space and build times?

Reply:
The CI jobs actually run faster than before, maybe the patch file triggered rebuilds anyway.

Reply:
We could define a `cargo make clean` maybe that cleans them up.